### DISCUSSION TITLE
No transmissions.xml

### DISCUSSION THREAD

**Discussion by TaliaKuznetsova** [Mogli12/KeyboardSteer#737] [Previous version]
So i prefer your setup compared to real manual transmission as you allow more than 3 ranges but I can't find a transmission.xml in my modsettings folder. Am i missing something or do i need to do something special to make it?
Edit: I haven't tried the developer version yet and therefore I am going to and will report on if that fixes it.
Edit 2: Nope. I can save my transmissions but i dont get a sheet like your examples.

**Comment by Mogli12** [comment]
The transmission are saved as individual files. transmissions.xml is deprecated.

**Comment by TaliaKuznetsova** [comment]
Ah ok thank you for clarification. On the subject of transmissions i've had an issue where i get a wheel physics line error and it shuts the transmission setting to off at random, usually when the rpm reaches the max. Is this a known issue or something i should make a formal report in a seperate post about.

**Comment by Mogli12** [comment]
Please post the complete log.txt and some screen shots. This should not happen.

**Comment by TaliaKuznetsova** [comment]
Been busy and hard to cause as its random so that's why I havent posted the issue yet. It's very inconsistent but it seems to occur quite often on long sessions so i need to play for a long time to see it in order to report on it.
